### PR TITLE
Story5

### PR DESCRIPTION
**Pull request by emferrari31**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -8,20 +8,34 @@ function shuffleArray(array) {
...
     playButton.remove()
     const checkAnswer = (button) => {
+
         button.addEventListener('click', () => {
             if (button.textContent === correctAnswer) {
                 button.style.background = '#04ac04'
-                //score++
+                score++
+                scoreBoxElem.textContent = score
+                console.log(score)
```
Comment (Henry-Tottle) [inline script.js:23]: delete console.log

File: style.css [hunk 2]
```diff
@@ -124,6 +124,28 @@ header {
...
 
+.scoreSection {
+    width: 100px;
+    height: 100px;
+}
+
+.timerDummy {
+    width: 100px;
+    height: 100px;
+}
```
Comment (Henry-Tottle) [inline style.css:135]: line space between selectors

File: style.css [hunk 3]
```diff
@@ -124,6 +124,28 @@ header {
...
+    /*height: 100px;*/
+    font-size: 2rem;
+    /*border: 1px solid black;*/
+    text-align: center;
+}
+
+.scoreLabel {
+    text-align: center;
+}
+
```
Comment (Henry-Tottle) [inline style.css:147]: one too many spaces

**Review by Henry-Tottle — COMMENTED** [review]
Looks good to me, few tiny tweaks before Matt mentions everything I missed :D

File: index.html [hunk 4]
```diff
@@ -32,6 +34,10 @@ <h1 class="appTitle">Film Quote Game</h1>
             </div>
             <div class="play">Play</div>
         </div>
+        <div class="scoreSection">
+            <p class="scoreLabel">Score:  </p>
```
Comment (matt-ioa) [inline index.html:38]: You will find that the space at the end of this text is removed in the HTML: it strips out any whitespace at the beginning and end of text automatically.

File: script.js (hunk 1 above)
Comment (matt-ioa) [reply script.js:23]: Why haven't you done this?!

File: script.js [hunk 5]
```diff
@@ -8,20 +8,33 @@ function shuffleArray(array) {
...
                 //trigger next question
 
             } else if (button.textContent === wrongAnswer1 || button.textContent === wrongAnswer2) {
-                button.style.background = 'red'
+                button.style.background = 'red' }
                 //trigger next question
-            }
-        })
+        }
+
```
Comment (matt-ioa) [inline script.js:36]: Remove this blank line

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Great job! Just a couple of very minor changes suggested, take a look

**Review by matt-ioa — APPROVED** [review]

**Pull request merged**